themes/startwords/assets/scss/_fonts.scss

```diff
@@ -1,127 +1,182 @@
+    font-display: swap;
+    src: url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-300italic.woff2') format('woff2'),
+         url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-300italic.woff') format('woff');
 }
 
+/* ibm-plex-serif-300 - latin */
 @font-face {
     font-family: 'IBM Plex Serif';
-    font-style: italic;
+    font-style: normal;
     font-weight: 300;
-    src: url('/fonts/IBM_Plex_Serif/IBMPlexSerif-LightItalic.ttf');
+    font-display: swap;
+    src: url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-300.woff2') format('woff2'),
+         url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-300.woff') format('woff');
 }
 
+/* ibm-plex-serif-regular - latin */
 @font-face {
     font-family: 'IBM Plex Serif';
     font-style: normal;
     font-weight: 400;
-    src: url('/fonts/IBM_Plex_Serif/IBMPlexSerif-Regular.ttf');
+    font-display: swap;
+    src: url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-regular.woff2') format('woff2'),
+         url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-regular.woff') format('woff');
 }
 
+/* ibm-plex-serif-italic - latin */
 @font-face {
     font-family: 'IBM Plex Serif';
     font-style: italic;
     font-weight: 400;
-    src: url('/fonts/IBM_Plex_Serif/IBMPlexSerif-Italic.ttf');
+    font-display: swap;
+    src: url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-italic.woff2') format('woff2'),
+         url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-italic.woff') format('woff');
 }
 
+/* ibm-plex-serif-500 - latin */
 @font-face {
     font-family: 'IBM Plex Serif';
     font-style: normal;
     font-weight: 500;
-    src: url('/fonts/IBM_Plex_Serif/IBMPlexSerif-Medium.ttf');
+    font-display: swap;
+    src: url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-500.woff2') format('woff2'),
+         url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-500.woff') format('woff');
 }
 
+/* ibm-plex-serif-500italic - latin */
 @font-face {
     font-family: 'IBM Plex Serif';
     font-style: italic;
     font-weight: 500;
-    src: url('/fonts/IBM_Plex_Serif/IBMPlexSerif-MediumItalic.ttf');
+    font-display: swap;
+    src: url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-500italic.woff2') format('woff2'),
+         url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-500italic.woff') format('woff');
 }
 
+/* ibm-plex-serif-700 - latin */
 @font-face {
     font-family: 'IBM Plex Serif';
     font-style: normal;
     font-weight: 700;
-    src: url('/fonts/IBM_Plex_Serif/IBMPlexSerif-Bold.ttf');
+    font-display: swap;
+    src: url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-700.woff2') format('woff2'),
+         url('/fonts/IBM_Plex_Serif/ibm-plex-serif-v8-latin-700.woff') format('woff');
 }
 
+/* ibm-plex-serif-700italic - latin */
 @font-face {
     font-family: 'IBM Plex Serif';
     font-style: italic;
     font-weight: 700;
-    src: url('/fonts/IBM_Plex_Serif/IBMPlexSerif-BoldItalic.ttf');
+    font-display: swap;
```

Comment:
Glad to learn about `font-display : swap`, have definitely seen the FOIT behavior on my phone on other sites.

Seems like the font declarations are very repetitive, wonder if it's worth a mixin? Although maybe not worth it now.

Reply:
Oh, that's a fantastic idea! I think we might use something very similar to this on other projects going forward, so I'd say it could be worth it. I'll experiment.

Reply:
On revisiting this, I'm not sure the best way to do it, since the things that change each `@font-face` declaration are the style, weight, and src and all are dependent in less-than-obvious ways on the _name_ of the font file (and which weights/styles we chose to include). There might be a way to standardize it, but it requires more thinking - maybe we can revisit for Geniza.